src/sentry/api/serializers/models/user.py

```diff
@@ -34,15 +46,17 @@ def get_attrs(self, item_list, user):
             )
         }
         identities = self._get_identities(item_list, user)
+        emails = self._get_useremails(item_list, user)
 
         authenticators = Authenticator.objects.bulk_users_have_2fa([i.id for i in item_list])
 
         data = {}
         for item in item_list:
             data[item] = {
                 'avatar': avatars.get(item.id),
-                'identities': identities.get(item.id),
+                'identities': identities[item.id],
```

Comment:
Ugh, kill me. Yeah. I'm going a bit too fast. And obviously not enough test coverage. I was just trying to unify behavior, but turns out, each one of these is actually a special snowflake. Sorry. 😞